api/routes/connection.ts

```diff
@@ -260,25 +274,41 @@ export default async function router(schema: Schema, config: Config) {
             await Auth.is_connection(config, req, {}, req.params.connectionid);
 
             if (await config.models.Layer.count({
-                where: sql`connection = ${req.params.connectionid}`
+                where: sql`connection =
+                ${req.params.connectionid}`
             }) > 0) throw new Err(400, null, 'Connection has active Layers - Delete layers before deleting Connection');
 
             if (await config.models.ConnectionSink.count({
-                where: sql`connection = ${req.params.connectionid}`
+                where: sql`connection =
+                ${req.params.connectionid}`
             }) > 0) throw new Err(400, null, 'Connection has active Sinks - Delete Sinks before deleting Connection');
 
             if (await config.models.Data.count({
-                where: sql`connection = ${req.params.connectionid}`
+                where: sql`connection =
+                ${req.params.connectionid}`
             }) > 0) throw new Err(400, null, 'Connection has active Data Syncs - Delete Syncs before deleting Connection');
 
             await config.models.Connection.delete(req.params.connectionid);
 
             await config.models.ConnectionToken.delete(sql`
-                connection = ${req.params.connectionid}
+                connection =
+                ${req.params.connectionid}
             `);
 
             config.conns.delete(req.params.connectionid);
 
+            const user = await Auth.as_user(config, req);
+            const profile = await config.models.Profile.from(user.email);
+
+            if (profile.id) {
```

Comment:
Can we add a check here to ensure the COTAK API is configured? 

Ref: https://github.com/dfpc-coe/CloudTAK/blob/main/api/routes/ldap.ts#L29